compose_testable_vignettes.R

```diff
@@ -46,9 +46,19 @@ rdt.cmd.head <- function(first, path)
     "include.configuration=TRUE,\n",
     "reload.state=", !first, ",\n",
     "block={\n\n",
+    "loadNamespace <- function(package, lib.loc = NULL,\n",
```

Comment:
`gcifno` should be inside the code block. We want to enable it, right before we begin tracing and disable it right after we are done tracing. More appropriate locations are `begin_trace` and `end_trace` hooks respectively.